pagopa/pn-bff
# Feature/PN-12922 - Notification cancellation document

## Short description

Updated micro-service dependencies

## List of changes proposed in this pull request

- Updated docs/openapi/common-refs.yaml

## How to test

Run a clean install -> run tests -> check that everything is ok

>[!WARNING]
>**IMPORTANT: remember to regenerate the external and aws files**
